base.js

```diff
@@ -0,0 +1,35 @@
+
+function renderErrors(errors) {
+	for (var i=0; i < errors.length; i++) {
+		$("#errors").append("<div> &raquo; " + errors[i] + "</div>").show();
+	};
+}
+
+function validateForm() {
+	result = true;
+	var errors = [];
+
+	if (document.getElementById("full-name").value == '') {
+		errors.push("Lauks 'Vārs, Uzvārds' ir jānorāda obligāti");
+		result = false;
+	}
+	if (document.getElementById("telefonanr").value == '') {
+		errors.push("Lauks 'Telefona numurs' ir jānorāda obligāti");
+		result = false;
+	}
+	if (document.getElementById("zinojums").value == '') {
+		errors.push("Lauks 'Ziņojums' ir jānorāda obligāti");
+		result = false;
+	}
+
+	if (errors[0] == undefined) {
+		return true;
+	} else {
+		renderErrors(errors);
+		return false;
+	}
+}
+
+window.onload = function() {
+	null;
+}
```

Comment:
General rule to add newline at the end of file to prevent any possible problems with Unix based tools